x-pack/plugins/security_solution/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -381,7 +386,7 @@ export const PolicyList = memo(() => {
           'Use policies to customize endpoint and cloud workload protections and other configurations',
       })}
     >
-      {totalItemCount > 0 ? (
+      {!shouldShowOnboarding ? (
```

Comment:
Could we add a test case for this? To ensure we display the error when there is an error.

Reply:
Added a test case to check when there is an api error. We have existing test cases to show the onboarding page when the list api returns no policies. I believe showing the error in the table is what we also do for the endpoint list. I know the other lists use toasts instead. I'm not sure if it's worth it to add a more descriptive toast additionally, but I could add one if you think it is?